independent-publisher-connectors/Fuxsy-AutodeskFusionManage/apiDefinition.swagger.json

```diff
@@ -0,0 +1,17161 @@
+                  urn:
+                    type: object
+                    properties:
+                      type:
+                        type: string
+                        example: string
+                  version:
+                    type: object
+                    properties:
+                      type:
+                        type: string
+                        example: integer
+              type:
+                type: string
+                example: object
+          examples:
+            application/json:
+              type: object
+              properties:
+                __self__:
+                  type: string
+                urn:
+                  type: string
+                uniqueName:
+                  type: string
+                displayName:
+                  type: string
+                scriptType:
+                  type: string
+                version:
+                  type: integer
+                dependsOn:
+                  type: array
+                  items:
+                    type: string
+                code:
+                  type: string
+        '401':
+          description: ''
+          schema:
+            $ref: '#/definitions/401'
+          examples:
+            application/json:
+              errorCode: SEC_UNAUTHORIZED
+              message: The request contains no authorization information.
+              params: ''
+              url: ''
+              errorClass: APIError
+        '404':
+          description: ''
+          schema:
+            $ref: '#/definitions/404_not_found'
+          examples:
+            application/json:
+              statusCode: 0
+              errors:
+                - string
+              message: string
+        '500':
+          description: ''
+          schema:
+            $ref: '#/definitions/500'
+          examples:
+            application/json:
+              statusCode: 0
+              message: string
+              activityId: string
+  /api/v3/workspaces/{WorkspaceID}/items/{ItemID}/views/10:
+    get:
+      description: >-
+        Retrieves a list of all relationship items associated with a specific
+        item.
+      operationId: Itemrelationships
+      summary: Item Relationships list
+      tags:
+        - Workspace/Item functions
+        - Workspace
+        - Item
+        - Views
+        - 10 (Relationships)
+      parameters:
+        - name: WorkspaceID
+          in: path
+          required: true
+          type: integer
+        - name: ItemID
+          in: path
+          required: true
+          type: integer
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: 200 OK
+          headers:
+            Cache-Control:
+              type: string
+            Connection:
+              type: string
+            Content-Encoding:
+              type: string
+            Content-Length:
+              type: string
+            Date:
+        
```

Comment:
Please correct the typo

Reply:
@vmanoharas I am not longer able to find this TYPO in new version swager file what I uploaded last time.
If there is still something can you please give me hint I can find that TYPO please?